airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/config.py

```diff
@@ -146,6 +172,10 @@ class BasicReadTestConfig(BaseConfig):
     expect_trace_message_on_failure: bool = Field(True, description="Ensure that a trace message is emitted when the connector crashes")
     timeout_seconds: int = timeout_seconds
     ignored_fields: Optional[Mapping[str, List[IgnoredFieldsConfiguration]]] = ignored_fields
+    file_types: Optional[FileTypesConfig] = Field(
+        default_factory=FileTypesConfig,
+        description="For file-based connectors, unsupported by source file types can be configured or a test can be skipped at all",
+    )
```

Comment:
I'm wondering if this should just be called `unsupported_file_types`, and should be a list/set (similar to `empty_streams`). WDYT?

Reply:
Hi @clnoll, thanks for the review!
I added this section as a complex object mostly because we need to have the ability to skip this test at all, and to do so, a user just need to set `FileTypesConfig.skip_test = True` instead of listing all available file types. Do you think there could be a better approach?

Reply:
Okay, I see what you mean now. LGTM!